docs/technical_reference/unit_models/stoichiometric_reactor.rst

```diff
@@ -0,0 +1,198 @@
+Stoichiometric reactor
+======================
+
+.. index::
+   pair: watertap.unit_models.stoichiometric_reactor;stoichiometric_reactor
+
+The main assumptions of the implemented model are as follows:
+
+1) Single liquid phase only (solids are not explicitly modeled)
+2) Steady state only
+3) Has no volume
+4) Reaction phenomena is not modeled (e.g. exothermic, endothermic, reaction rate, etc.)
+   User must provide amount of reagent to add, and amount of precipitant that would form
+
+Introduction
+------------
+The stoichiometric reactor is a basic unit operation designed to aid in modeling 
+addition of reagent to a feed stream and removal of ions from a stream through precipitation.
+The most basic usage is for modeling Lime/Soda ash softening process. 
```

Comment:
This is more of a "basic **example**" rather than "basic usage"
```suggestion
A basic example for using this model is the lime/soda ash softening process. 
```